backend/api/admin.py

```diff
@@ -271,6 +282,25 @@ class SpectroConfigAdmin(admin.ModelAdmin):
         "window_size",
         "overlap",
         "zoom_level",
+        "desc",
+        "spectro_normalization",
+        "data_normalization",
+        "zscore_duration",
+        "HPfilter_min_freq",
+        "colormap",
+        "dynamic_min",
+        "dynamic_max",
+        "window_type",
+        "frequency_resolution",
+        "time_resolution_zoom_0",
+        "time_resolution_zoom_1",
+        "time_resolution_zoom_2",
+        "time_resolution_zoom_3",
+        "time_resolution_zoom_4",
+        "time_resolution_zoom_5",
+        "time_resolution_zoom_6",
+        "time_resolution_zoom_7",
+        "time_resolution_zoom_8",
```

Comment:
What's with those 8 new fields :sweat_smile: ? What'd be good is some specifications, if this was decided on Slack copy-pasting the discussion in the issue description would be great :+1: It's very hard to suggest possible improvements without knowing what the the original intent :wink: